test/downstream/symbol_indexing.jl

```diff
@@ -363,13 +365,13 @@ sol = solve(prob, Tsit5())
     p = [a => 1.0
          b => 100.0]
     prob = OptimizationProblem(sys, u0, p, grad = true, hess = true)
-    sol = solve(prob, GradientDescent())
-    @test sol[x]≈1 atol=1e-3
-    @test sol[y]≈1 atol=1e-3
-    @test getp(sys, a)(sol) ≈ 1
-    @test getp(sys, b)(sol) ≈ 100
-    @test sol.ps[a] ≈ 1
-    @test sol.ps[b] ≈ 100
+    sol = @test_throws ArgumentError solve(prob, GradientDescent())
```

Comment:
what's the error here?

Reply:
@Vaibhavdixit02

Reply:
```
ERROR: ArgumentError: Fminbox(GradientDescent{LineSearches.InitialPrevious{Float64}, LineSearches.HagerZhang{Float64, Base.RefValue{Bool}}, Nothing, Optim.var"#13#15"}(LineSearches.InitialPrevious{Float64}
  alpha: Float64 1.0
  alphamin: Float64 0.0
  alphamax: Float64 Inf
, LineSearches.HagerZhang{Float64, Base.RefValue{Bool}}
  delta: Float64 0.1
  sigma: Float64 0.9
  alphamax: Float64 Inf
  rho: Float64 5.0
  epsilon: Float64 1.0e-6
  gamma: Float64 0.66
  linesearchmax: Int64 50
  psi3: Float64 0.1
  display: Int64 0
  mayterminate: Base.RefValue{Bool}
, nothing, Optim.var"#13#15"(), Flat())) requires gradients, since you didn't use `OptimizationFunction` with a valid AD backend https://docs.sciml.ai/Optimization/stable/API/ad/ the lower and upper bounds thus will be ignored.
```
I had messaged Vaibhav about it when it came up, not sure if he was able to fix it yet.

Reply:
It was fixed I have created a release for it